runtime/mainnet/src/config/collation.rs

```diff
@@ -0,0 +1,288 @@
+use pop_runtime_common::{HOURS, SLOT_DURATION};
+
+use crate::{
+	parameter_types, AccountId, Aura, AuraId, Balances, CollatorSelection, ConstBool, ConstU32,
+	ConstU64, EnsureRoot, PalletId, Runtime, RuntimeEvent, Session, SessionKeys,
+};
+
+impl pallet_authorship::Config for Runtime {
+	type EventHandler = (CollatorSelection,);
+	type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Aura>;
+}
+
+parameter_types! {
+	pub const Period: u32 = 6 * HOURS;
+}
+
+#[docify::export(aura_config)]
+impl pallet_aura::Config for Runtime {
+	type AllowMultipleBlocksPerSlot = ConstBool<true>;
+	type AuthorityId = AuraId;
+	type DisabledValidators = ();
+	// Number of blocks we can produce per session.
+	type MaxAuthorities = Period;
+	type SlotDuration = ConstU64<SLOT_DURATION>;
+}
+
+parameter_types! {
+	pub const PotId: PalletId = PalletId(*b"PotStake");
+	pub const Offset: u32 = 0;
+}
+
+/// We allow root to execute privileged collator selection operations.
+pub type CollatorSelectionUpdateOrigin = EnsureRoot<AccountId>;
+
+impl pallet_collator_selection::Config for Runtime {
+	type Currency = Balances;
+	// should be a multiple of session or things will get inconsistent
+	type KickThreshold = Period;
+	type MaxCandidates = ConstU32<0>;
+	type MaxInvulnerables = ConstU32<20>;
+	type MinEligibleCollators = ConstU32<3>;
+	type PotId = PotId;
+	type RuntimeEvent = RuntimeEvent;
+	type UpdateOrigin = CollatorSelectionUpdateOrigin;
+	type ValidatorId = <Self as frame_system::Config>::AccountId;
```

Comment:
```suggestion
	type ValidatorId = AccountId;
```

any reason not to do this? It compiles.

Reply:
Good one! Thanks

Reply:
[be40c5f](https://github.com/r0gue-io/pop-node/pull/439/commits/be40c5fe8a7efc1bde595971b08a6a61e78ab9ad)